src/v/config/configuration.cc

```diff
@@ -67,6 +68,13 @@ configuration::configuration()
       "IpAddress and port for supervisor service",
       required::no,
       unresolved_address("127.0.0.1", 43189))
+  , coproc_registration_broker(
```

Comment:
what? no. the kafka client does this dynamically. @BenPope can show you.

Reply:
This looks ok to me. The client needs the list of seed brokers (as `unresolved_address`), will grab metadata from the first that succeeds, and then connect to the set of brokers based on that.

Reply:
sorry, i meant to say we should always use this node's listen addr really. 

how could coproc know what a registration broker is... this machine *is* the registration broker.